### PR TITLE
Updates butte trap to exclude fyke only days

### PR DESCRIPTION
**Pull request by ErinCain**
Update butte creek data in standard format datasets - I made the following changes: 

- Updated subsite names so we didn't have "okie fyke trap" and "okie dam fyke trap" - changed all to "okie dam fyke trap" 
- Updated all butte datasets to remove days that only fyke trap sampling 

Additionally I was getting and error with the mutate statement ifelse( , ,`NA_real_`) so I changed `NA_real_` to `NA` 

I also was having some speed issues with the catch script. I found that adding an ungroup() after the summarize statements improved performance drastically.

### CODE REVIEW

File: data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd [hunk 1]
```diff
@@ -400,28 +401,36 @@ butte_trap_pre_2015 <- butte_rst %>%
...
-butte_trap_combined <- bind_rows(butte_trap_pre_2015, butte_trap_camp_dates) |>  rename(trap_min = min)
+butte_trap_combined <- bind_rows(butte_trap_pre_2015, butte_trap_camp_dates) %>%  rename(trap_min = min)
 
 # min date in catch and trap aligns 
 butte_min_date_compare <- full_join(butte_rst_min, butte_trap_combined)
 ```
 
 
 ```{r, final_butte_edits}
+# find all dates that a trap that is not okie dam fyke trap is running
```
Comment (alvizek) [inline data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd:419]: I like the way you did the filtering here. This assumes that the NA subsites are not fyke sites which I think is the right call. I debated about adding a subsite label to the NA subsites but ended up deciding that it is more clear to leave as NA.

File: data-raw/standard-format-data-prep/mark_recapture_standard_format.Rmd [hunk 2]
```diff
@@ -170,7 +170,7 @@ butte_released <- read_csv(here::here("data", "rst", "butte_releases.csv")) %>%
          run_released = ifelse(is.na(run_released), "not recorded", tolower(run_released)),
```
Comment (alvizek) [inline data-raw/standard-format-data-prep/mark_recapture_standard_format.Rmd:170]: In the butte_recaptured table can we make the same change to "okie dam fyke trap"? I think right now it is still just "okie fyke trap"

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd [hunk 3]
```diff
@@ -236,21 +235,21 @@ butte_trap_clean_pre_2015 <- butte_rst %>%
   select(-gear_id)
```
Comment (alvizek) [inline data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:237]: When the NA_real_ were changed to NA this no longer worked for me. Is it working for you?

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd [hunk 4]
```diff
@@ -397,7 +405,7 @@ deer_trap_clean <- deer_rst %>%
          site = stream,
```
Comment (alvizek) [inline data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:405]: This isn't working for me anymore either.

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd [hunk 5]
```diff
@@ -447,13 +455,13 @@ feather_trap_clean <- feather_trap_camp %>%
          trap_visit_date = as.Date(visitTime),
          trap_visit_time = as_hms(visitTime),
          trap_stop_date = case_when(visitType %in% c("Continue trapping", "Unplanned restart", "End trapping") ~ as.Date(visitTime),
```
Comment (alvizek) [inline data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:457]: same here. not working for me

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd [hunk 6]
```diff
@@ -521,7 +529,7 @@ mill_trap_clean <- mill_rst %>%
          site = stream,
```
Comment (alvizek) [inline data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:529]: same here. not working

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd [hunk 7]
```diff
@@ -614,13 +622,13 @@ knights_trap_clean_2006 <- knights_effort %>%
          trap_visit_date = as.Date(visitTime),
```
Comment (alvizek) [inline data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:622]: same here. not working

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd [hunk 8]
```diff
@@ -735,13 +743,13 @@ tisdale_trap_clean <- tisdale_trap_camp %>%
...
```
Comment (alvizek) [inline data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:743]: same here. not working

**Review by alvizek — CHANGES_REQUESTED** [review]
I'm not sure if this is just a me problem but when you changed the NA_real_ to NA the code errors out. I want to make sure it is working for you and then I can figure out what my issue is.

I included just a few other small comments.

File: data-raw/standard-format-data-prep/mark_recapture_standard_format.Rmd (hunk 2 above)
Comment (ErinCain) [reply data-raw/standard-format-data-prep/mark_recapture_standard_format.Rmd:170]: updated

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd (hunk 3 above)
Comment (ErinCain) [reply data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:237]: Updated all the NAs within case_when() statements to use as.Date(NA) or as_hms(NA) or as.character(NA) or as.logical(NA) depending on data types.

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd (hunk 4 above)
Comment (ErinCain) [reply data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:406]: updated

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd (hunk 5 above)
Comment (ErinCain) [reply data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:458]: updated

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd (hunk 6 above)
Comment (ErinCain) [reply data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:530]: updated

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd (hunk 7 above)
Comment (ErinCain) [reply data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:623]: updated

File: data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd (hunk 8 above)
Comment (ErinCain) [reply data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd:744]: updated

**Review by alvizek — APPROVED** [review]

**Pull request merged**